interpreter/cling/include/cling/std_darwin.modulemap

```diff
@@ -0,0 +1,2064 @@
+module std [system] {
+export *
```

Comment:
The rest is copy-paste from the original modulemap file. We can teach cmake to take the contents and put them here if necessary...

Reply:
What is 'original' in this case?

Reply:
The one that comes with xcode.

Reply:
If comments are allowed, could we add that info as the first line?